test/main/deepEqual.js

```diff
@@ -1581,6 +1581,39 @@ QUnit.test( "Test that must be done at the end because they extend some primitiv
 	}
 );
 
+QUnit.test( "Compare Number values", function( assert ) {
```

Comment:
@mixed: would you create a single module on the top of these tests called "Compare primitive values"?

Example:

```js
QUnit.module( "Compare primitive values" );
```